R/mouse/mouse_cell_type_mapping.R

```diff
@@ -0,0 +1,131 @@
+#' Functions for mapping cell types between deconvolution methods.
+
+require(tidyverse)
+require(magrittr)
+require(rlang)
+require(ComICS)
```

Comment:
In an R package, you should never use `require` or `library`. It is better to use `@importFrom` to only import those methods that are required. 

See for instance
https://github.com/icbi-lab/immunedeconv/blob/c70539f2b08901687561dca755337fc6a5130440/R/immune_deconvolution_methods.R#L1-L18